components/library/SDL2_image/SDL2_image.p5m

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Till Wegmueller
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/include/SDL2/SDL_image.h
+file path=usr/lib/$(MACH64)/cmake/SDL2_image/sdl2_image-config-version.cmake
+file path=usr/lib/$(MACH64)/cmake/SDL2_image/sdl2_image-config.cmake
+link path=usr/lib/$(MACH64)/libSDL2_image-2.0.so.0 \
+    target=libSDL2_image-2.0.so.0.800.2
+file path=usr/lib/$(MACH64)/libSDL2_image-2.0.so.0.800.2
+file path=usr/lib/$(MACH64)/libSDL2_image.a
+link path=usr/lib/$(MACH64)/libSDL2_image.so target=libSDL2_image-2.0.so.0.800.2
```

Comment:
With shared libs available we usually don't deliver static libs. Is there a reason that we want it here?

Reply:
Not really. Dropped